src/style.css

```diff
@@ -0,0 +1,132 @@
+.page-html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  color: #060b35;
+}
+
+.reset {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  box-sizing: border-box;
+  width: 200px;
+  min-height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__img {
+  padding-top: 32px;
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__name {
+  padding: 40px 16px 0;
+  width: 100%;
+
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+}
+
+.product-card__code {
+  padding-left: 16px;
+  flex-basis: 100%;
+  padding-top: 4px;
+  width: 98px;
+
+  line-height: 14px;
+  font-size: 10px;
+
+  color: #616070;
+}
+
+.stars-container {
+  display: flex;
+  flex-basis: 100%;
+  justify-content: space-between;
+  padding: 16px;
+}
+
+.stars-container__stars {
+  display: flex;
+  padding-right: 17px;
+}
+
+.stars-container__stars-star {
+  content: " ";
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+
+  background-position: 50%;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars-container__reviews {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  line-height: 14px;
+  font-size: 10px;
+}
+
+.stars-container__stars > .stars-container__stars-star:last-child {
+  background-image: url(./images/star.svg);
+}
+
+.product-card__price {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 8px 16px 0;
+
+  line-height: 18px;
+}
+
+.product-card__price-text {
+  color: #616070;
+}
+
+.product-card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.product-card__buy {
+  margin-top: 16px;
+  border-radius: 5px;
+  border: 1px solid transparent;
+  width: 166px;
+  height: 40px;
+  box-sizing: border-box;
+  
+  background: #00acdc;
+  color: #fff;
+
+  line-height: 38px;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.product-card__buy:hover {
+  width: 164px;
+  height: 38px;
+
```

Comment:
don't use width, height for hover. Only change color, background-color, border-color